students/barecki_andrzej/lesson_01_basics/lesson_01_basics_all_cases.py

```diff
@@ -0,0 +1,118 @@
+# Lesson 1 (Input, print and numbers)
+
+print('Problem 1: Sum of three numbers')
+print('Set first number:')
+a = int(input())
+print('Set second number:')
+b = int(input())
+print('Set third number:')
+c = int(input())
+print(a + b + c)
+
+print('\n\rProblem 2: Area of right-angled triangle')
+print('Set length of the base of a right-angled triangle')
+b = int(input())
+print('Set length of the height of a right-angled triangle')
+h = int(input())
+print('The area of right-angled triangle is equal: ' + str((1/2)*b*h))
+
+print('\n\rProblem 3: Hello, Harry!')
+print('Set user name: ')
+name = input()
+print('Hello, ' + str(name) + '!')
+
+print('\n\rProblem 4: Apple sharing')
+print('Set number of students:')
+N = int(input())
+print('Set number of apples:')
+K = int(input())
+print('Each student get ' + str(K // N) + ' apples')
+print('In basket remains ' + str(K % N) + ' apples')
+
+print('\n\rProblem 5: Previous and next')
+print('Set an integer:')
+a = int(input())
+print('The next number for the number ' + str(a) + ' is ' + str(a + 1) + '.')
+print('The previous number for the number ' + str(a) + ' is ' + str(a - 1) + '.')
+
+print('\n\rProblem 6: School desks')
+def compute_desks( nr ):
```

Comment:
wow, funkcja :)
tego już nie poprawiaj, bo funkcje to materiał kolejnych lekcji, ale poczytaj o whitespaceach i newlineach w PEP8 (link jest poniżej) i blank lineach przy definicji funkcji: https://www.python.org/dev/peps/pep-0008/#blank-lines

Reply:
Poprawilem zgodnie z sugestiami PyCharm.